site/styles/_forms.scss

```diff
@@ -116,6 +139,9 @@ s72-activatedevice-form,
       }
     }
   }
+  .s72-form-control {
+    height: calc(1.5em + 0.75rem + 4px); // Selectable Inputs (birthdate/gender were too short on mobile)
```

Comment:
This calculation is done three times - perhaps it's worth defining as a variable and explaining what the values are representing.

Reply:
@cunhapaola @PeculiarGoat Sorry, I know this just merged, but did you explain this?

Reply:
@stajs calc is gone... this thread is outdated!

Reply:
Wonderful 🌹